packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -2161,7 +2161,9 @@ describeCompat(
 			},
 		);
 
-		itExpects(
+		// AB#14900, 20297: this test is extremely flaky on Tinylicious and causing noise.
+		// Skip it for now until above items are resolved.
+		itExpects.skip(
```

Comment:
Since it's flaky for tinylicious only, it should probably be disabled only for tinylicious so we get coverage for other services.